src/test/java/pl/coderslab/service/CargoServiceTest.java

```diff
@@ -54,11 +56,26 @@ public void shouldUpdateIfExist() {
         cargo.setName("melasa");
 
         cargoService.update(cargo);
-        Cargo cargo1=cargoService.showById(100L).orElseThrow();
+        Cargo cargo1=cargoService.showById(100L);
         Assertions.assertThat(cargo1.getName()).isEqualTo("melasa");
 
     }
 
+    @Test(expected = RuntimeException.class)
```

Comment:
Fajnie, że podzieliłes kod na 3 sekcje jednak, trochę nieprawidłowo: 
given - to sekcja gdzie są warunki przed testem
when - to to co testujemy 
then - to czego się spoodziewamy w twoim przypadku then nie będzie bo spodziewamy sie wyjątku